.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    env: {
```

Comment:
indent and single quotes.  Is there some bad version of this file somewhere being used as a template?

Reply:
comes from npx eslint --init
- [ ] format the eslintrc.js file

Reply:
@mattcollier try again when ready.

Reply:
btw we really might want to store gists of good ones somewhere so when we add eslint to a repo we have something to use.

Reply:
i think having templates in the eslint-config-digitalbazaar repo seems appropriate.

Reply:
if you haven't done it or created an issue yet I will. other wise feel free to.

Reply:
not it.